src/ol/renderer/canvas/canvastilelayerrenderer.js

```diff
@@ -365,7 +365,7 @@ ol.renderer.canvas.TileLayer.prototype.prepareFrame =
   /** @type {Array.<number>} */
   var zs = Object.keys(tilesToDrawByZ).map(Number);
   zs.sort(ol.array.numberSafeCompareFunction);
-  var opaque = tileSource.getOpaque();
+  var opaque = tileSource.getOpaque(projection);
```

Comment:
The `source.getOpaque(projection)` method is now a bit hard to explain.  And the `opaque` value is deceptive.  Why not move the projection equivalence check here?  Maybe something like:
```js
var opaque = tileSource.getOpaque();
var opaqueAndNotReprojecting = opaque &&
    ol.ENABLE_RASTER_REPROJECTION &&
    ol.proj.equivalent(tileSource.getProjection(), projection);
```
And then use `opaqueAndNotReprojecting` below.  Cumbersome name, but at least it describes what it is.